modules/nf-core/openms/idscoreswitcher/main.nf

```diff
@@ -0,0 +1,49 @@
+process OPENMS_IDSCORESWITCHER {
+    tag "$meta.id"
+    label 'process_single'
+
+    conda "${moduleDir}/environment.yml"
+    container "${ workflow.containerEngine == 'singularity' && !task.ext.singularity_pull_docker_container ?
+        'https://depot.galaxyproject.org/singularity/openms:3.1.0--h8964181_3':
+        'biocontainers/openms:3.1.0--h8964181_3' }"
+
+    input:
+    tuple val(meta), path(idxml)
+
+    output:
+    tuple val(meta), path("*.idXML"), emit: idxml
+    path "versions.yml"             , emit: versions
+
+    when:
+    task.ext.when == null || task.ext.when
+
+    script:
+    def args = task.ext.args ?: ''
+    def prefix = task.ext.prefix ?: "${meta.id}_score_switched"
```

Comment:
Is `_score_switched` necessary here?

Reply:
Thanks for the review :) Not necessary I guess, but there is the risk of overwriting the idxml file in the workdir if`meta.id == idxml.baseName`. Should the developer be aware of this?

Reply:
You're welcome! Most modules write some kind of check to prevent that. See [here](https://nf-co.re/docs/contributing/modules#naming-conventions) for the guidelines on that.

Reply:
Ah that makes sense, thanks for the hint. I added a check according to the guidelines